Mathlib/Tactic/Rewrites.lean

```diff
@@ -181,7 +187,7 @@ def rewritesCore (lemmas : DiscrTree (Name × Bool × Nat) s × DiscrTree (Name
 def rewrites (lemmas : DiscrTree (Name × Bool × Nat) s × DiscrTree (Name × Bool × Nat) s)
     (goal : MVarId) (target : Expr) (stop_at_rfl : Bool := False) (max : Nat := 20)
```

Comment:
Minor, but is there a reason that this is of type `Bool` but by default a (coerced) `False : Prop` (and snake-cased like a proof instead of a `Bool`)?

Reply:
Oops, good catches. I'll fix these here.